index.js

```diff
@@ -30,32 +30,47 @@ module.exports = {
   treeForPublic(tree) {
     let trees = [];
     const targetBrand = this.options['@embroider/macros'].setOwnConfig.brand;
-    const pkgName = this.parent.pkg.name;
-    const isEngine = checkIfEngine(this.parent.pkg);
+    const ebmCandidates = this.parent.addons
+      .map((addon) => addon.addons)
+      .flat()
+      .filter((a) => a.addons.find((addon) => addon.pkg.name.includes('@upfluence/ember-brand-manager')))
+      .reduce(
+        (acc, v) => {
+          console.log('<< fo', v.name, v.pkg.root);
+          acc.push({ pkgName: v.name, isEngine: true, srcDir: v.pkg.root });
+          return acc;
+        },
+        [{ pkgName: this.parent.pkg.name, isEngine: checkIfEngine(this.parent.pkg) }]
+      );
 
-    debugLog(`\n[EBM] Target brand : [${targetBrand}]`);
-    debugLog(`[EBM] Parent pkg name : [${pkgName}]`);
```

Comment:
any reason this debugLog has been removed ?

Reply:
it's been moved here: https://github.com/upfluence/ember-brand-manager/pull/7/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R54